src/Controls/tests/DeviceTests/Elements/Toolbar/ToolbarTests.cs

```diff
@@ -119,27 +119,27 @@ public async Task ToolbarTitle()
 
 		[Theory]
 		[InlineData($"{nameof(FlyoutPage)}WithNavigationPage, {nameof(ContentPage)}, {nameof(FlyoutPage)}WithNavigationPage"
-#if ANDROID || WINDOWS
+#if WINDOWS
```

Comment:
If ANDROID is removed from the check, should the Skip comment also be updated?
For eg: "Currently Failing on Windows https://github.com/dotnet/maui/issues/17444"?